scripts/jenkins-yoctobuild-build.sh

```diff
@@ -398,8 +398,19 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
     # Use release tool to query for available docker names.
     for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker -a ); do (
 
-        git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
-        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
+        if $WORKSPACE/integration/extra/release_tool.py --help | grep -e '--map-name FROM-TYPE SERVICE TO-TYPE'; then
+            git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
+            docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
+        else
+            case "$docker" in
+                mender-client-docker) git="mender";;
+                api-gateway) git="mender-api-gateway-docker";;
```

Comment:
Unrelated: But isn't it a little weird some of them are named Mender, and others are not?

Reply:
It is the name of the repo, nothing we can do about it now.